app/src/main/java/com/example/concordia_campus_guide/Global/ApplicationState.java

```diff
@@ -18,6 +18,7 @@
     private Buildings buildings;
```

Comment:
Didn't we say that we won't need the ApplicationState anymore?

Reply:
we still need application state to load the json